config/system/default.yaml

```diff
@@ -32,7 +32,7 @@ site_path: local/site/
 
 # DHCP lease time parameters
 initial_dhcp_lease_time: 120s
-dhcp_lease_time: 500s
+dhcp_lease_time: 120s
```

Comment:
this doesn't seem right... the ongoing lease time shouldn't be so short (not realistic behavior). What's the impetus behind this change?

Reply:
@grafnu Simply to reduce the overall time that DAQ takes to run. 500s is fine but to be most accurate with the DHCP lease time wait test, a wait (equivalent to the total DHCP lease time) should be imposed to ensure a complete DHCP lease exipiry occurs. Therefore, without this change, DAQ will take 500s on this single test.

Reply:
If this is just for one test then it should be applied only to that one test and not as a blanket global.  The 500s is there for a reason since it's more typical behavior. And this file is not only for testing.